neqo-transport/tests/server.rs

```diff
@@ -56,6 +56,47 @@ pub fn complete_connection(
     connected_server(server)
 }
 
+#[test]
+fn connect_to_nothing() {
+    // Start a server to get the token that will be used for calling
+    // `client.enable_resumption`.
+    let mut server = default_server();
+    let token = generate_ticket(&mut server);
+
+    // Discharge the old connection so that we don't have to worry about it.
+    let mut now = now();
+    let t = server.process(None, now).callback();
+    now += t;
+
+    let mut client = default_client();
+    // After enable_resumption, the client have the stream limit settings from
+    // the server, so we can create a stream.
+    client.enable_resumption(now, &token).unwrap();
+
+    let client_stream = client.stream_create(StreamType::BiDi).unwrap();
+    client.stream_keep_alive(client_stream, true).unwrap();
+    let mut client_send = |now| {
+        client.stream_send(client_stream, &[0x0; 500]).unwrap();
+        match client.process(None, now) {
```

Comment:
Does this change anything?

```suggestion
        match client.process_output(now) {
```

Reply:
No, same result ('earliest > now' assertion).

Reply:
Good.  Just confirming.  I generally prefer not to use `process(None, ...)` in place of `process_output(...)`.